### PR TITLE
[DSD-2918] Updated mosip-modules-deployment.md

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

**Review by ckm007 — APPROVED** [review]

File: docs/deploymentnew/mosip-modules-deployment.md [hunk 1]
```diff
@@ -186,5 +193,7 @@ cd $INFRA_ROOT/deployment/v3/mosip/resident
 
 ```
 cd $INFRA_ROOT/deployment/v3/mosip/regclient
+sudo apt-get update
```
Comment (ckm007) [inline docs/deploymentnew/mosip-modules-deployment.md:196]: -y is missing in both

File: docs/deploymentnew/mosip-modules-deployment.md [hunk 2]
```diff
@@ -7,24 +7,25 @@ Below is the sequence of installation of MOSIP modules and the sequence must be
...
-17. [Admin](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/admin/README.md)
-18. [ID Authentication](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/ida/README.md)
-19. [Print](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/print/README.md)
-20. [Partner Onboarder](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/partner-onboarder/README.md)
-21. [MOSIP File Server](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/mosip-file-server/README.md)
-22. [Registration Client](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/regclient/README.md)
-23. [Resident Services](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/resident/README.md)
+6. [Mock SMTP](https://github.com/mosip/mosip-infra/blob/develop/deployment/v3/mosip/mock-smtp/README.md)
+7. [Kernel](https://github.com/mosip/mosip-infra/blob/1.2.0.1/deployment/v3/mosip/kernel/README.md)
+8. [Masterdata-loader]()
```
Comment (SwatiPeriwal) [inline docs/deploymentnew/mosip-modules-deployment.md:12]: link missing

**Review by SwatiPeriwal — APPROVED** [review]
Approving changes.... master-loader link missing.. pls check and update

File: docs/deploymentnew/mosip-modules-deployment.md (hunk 1 above)
Comment (Mohanraj209) [reply docs/deploymentnew/mosip-modules-deployment.md:196]: Updated in the install.sh script sir.

File: docs/deploymentnew/mosip-modules-deployment.md (hunk 2 above)
Comment (Mohanraj209) [reply docs/deploymentnew/mosip-modules-deployment.md:12]: As we don't have README.md file for masterdata-loader so the link is not present.

**Pull request merged**